src/authorization/AuxiliaryAuthorizer.ts

```diff
@@ -0,0 +1,50 @@
+import type { AuxiliaryIdentifierStrategy } from '../ldp/auxiliary/AuxiliaryIdentifierStrategy';
+import { getLoggerFor } from '../logging/LogUtil';
+import { NotImplementedHttpError } from '../util/errors/NotImplementedHttpError';
+import type { AuthorizerArgs } from './Authorizer';
+import { Authorizer } from './Authorizer';
+
+/**
+ * An authorizer for auxiliary resources such as acl or shape resources.
+ * The access permissions of an auxiliary resource depend on those of the resource it is associated with.
+ * This authorizer calls the source authorizer with the identifier of the associated resource.
+ */
+export class AuxiliaryAuthorizer extends Authorizer {
+  protected readonly logger = getLoggerFor(this);
+
+  private readonly resourceAuthorizer: Authorizer;
+  private readonly strategy: AuxiliaryIdentifierStrategy;
```

Comment:
`strategy` is correct here since this authorizer handles all auxiliary resources. I do notice in the configs that currently it is linked to the acl strategy (which is sort of correct since it's the only one we have), but I should already link it to the composite one for when future auxiliary types get added.

Reply:
I probably meant `auxStrategy`. Just found it weird to read `strategy` in the code below; like, what strategy?